crates/ra_project_model/src/cargo_workspace.rs

```diff
@@ -249,3 +252,10 @@ impl CargoWorkspace {
         &self.workspace_root
     }
 }
+
+#[test]
+fn test_cargo_metadata() {
+    let cargo_toml = PathBuf::from("/");
+    let result = CargoWorkspace::from_cargo_metadata(&cargo_toml, &CargoFeatures::default());
+    println!("error: {:?}", result)
+}
```

Comment:
This test doesn't assert anything and probably should be removed?

Reply:
Oops - forgot to take it out.